deps/chakrashim/src/v8objecttemplate.cc

```diff
@@ -748,6 +748,20 @@ Local<Object> ObjectTemplate::NewInstance(Handle<Object> prototype) {
     }
   }
 
+  if (!objectTemplateData->className.IsEmpty()) {
+    jsrt::IsolateShim* iso = jsrt::IsolateShim::GetCurrent();
+    if (jsrt::DefineProperty(newInstanceRef,
+			     iso->GetToStringTagSymbolPropertyIdRef(),
+			     jsrt::PropertyDescriptorOptionValues::True, /* writable */
+			     jsrt::PropertyDescriptorOptionValues::False, /* enumerable */
+			     jsrt::PropertyDescriptorOptionValues::True, /* configurable */
+			     *objectTemplateData->className,
+			     JS_INVALID_REFERENCE,
+			     JS_INVALID_REFERENCE) != JsNoError) {
```

Comment:
Too much (or too little) indentation here.  Either line everything up under `newInstanceRef` or move `newInstanceRef` down to a new line and indent everything twice.